src/Pact/Types/Capability.hs

```diff
@@ -69,17 +69,27 @@ instance NFData SigCapability
 instance Pretty SigCapability where
   pretty SigCapability{..} = parens $ hsep (pretty _scName:map pretty _scArgs)
 
+sigCapabilityProperties :: JsonProperties SigCapability
+sigCapabilityProperties o =
+  [ "args" .= _scArgs o
+  , "name" .= _scName o
+  ]
+{-# INLINE sigCapabilityProperties #-}
+
 instance ToJSON SigCapability where
-  toJSON (SigCapability n args) = object $
-    [ "name" .=  n
-    , "args" .= args
-    ]
+  toJSON = enableToJSON "Pact.Types.Capability.SigCapability" . object . sigCapabilityProperties
```

Comment:
I think, I would rather leave them in the code for future debugging and testing. It was just too useful to be able to switch that on when trying to find out where exactly some code would call `toJSON` while working on this PR.

For now it is just an inlined `id`. So, there's no perf impact. Eventually, if we fully get rid of `toJSON`,  it would become an error.